packages/wp-story-editor/src/api/media.js

```diff
@@ -188,7 +188,7 @@ export function updateMedia(config, mediaId, data) {
     path: `${config.api.media}${mediaId}/`,
     data,
     method: 'POST',
-  });
+  }).then(getResourceFromAttachment);
```

Comment:
Should we update `deleteMedia` below to return `void`?

See previous discussion at https://github.com/google/web-stories-wp/pull/9683/files#r758728852